app/controllers/admin/basic_projects_controller.rb

```diff
@@ -20,4 +20,50 @@ def index
       @csv_mode = false
     end
   end
+
+  def show
+    @project = BasicProject.find(params[:id])
+    authorize @project
+    prep_form_vars
+  end
+
+  def destroy
+    @project = BasicProject.find(params[:id])
+    authorize @project
+
+    if @project.destroy
+      redirect_to admin_basic_projects_path, notice: I18n.t(:notice_deleted)
+    else
+      prep_form_vars
+      render :show
+    end
+  end
+
+  def update
+    @project = BasicProject.find(params[:id])
+    authorize @project
+    @project.assign_attributes(basic_project_params)
+
+    if @project.save
+      redirect_to admin_basic_project_path(@project), notice: I18n.t(:notice_updated)
+    else
+      prep_form_vars
+      render :show
+    end
+  end
+
+  private
+
+  def prep_form_vars
+    @agent_choices = person_policy_scope(Person.in_division(selected_division).where(has_system_access: true)).order(:name)
```

Comment:
I fixed this one. As we are at end of sprint, I am going to address these others in a different pull request. That way I can test and make sure all is working well rather than rushing that change in. Thanks, @hooverlunch.